ParallelPrimitives/RadixSort.cpp

```diff
@@ -18,6 +18,9 @@
 #include <dlfcn.h>
 #endif
 
+inline uint64_t div_round_up64( uint64_t val, uint64_t divisor ) { return ( val + divisor - 1 ) / divisor; }
```

Comment:
I'm not sure if it really helps something in this case. but I just put it

Reply:
A small explanation:

What I meant is this: https://godbolt.org/z/43vqYnPfn

Using `constexpr` whenever possible and feasible in the code will help detect error or UB at compile time.

In this example, `divisor` will result in zero, so the code will not compile. In a real-world example, sometimes it might not be as obvious as what's shown there.  This will not be the case if everything is evaluated at runtime. Therefore, generally it is recommended to mark functions as `constexpr` .